D:\Python>python -m doctest -v -o NORMALIZE_WHITESPACE issue_01.py
Trying:
    encode('SOS')
Expecting:
    '... --- ...'
ok
Trying:
    encode('HIGHLOAD')
Expecting:
    '.... .. --. .... .-.. --- .- -..'
ok
Trying:
    encode('HOW MANY TIMES I SLEPT?')
Expecting:
    '.... --- .-- -- .- -. -.-- - ..
    -- . ... .. ... .-.. . .--. - ..--..'
ok
Trying:
    encode('Аннушка уже разлила масло...') #doctest: +ELLIPSIS
Expecting:
    '.-.-.- .-.-.-'
**********************************************************************
File "D:\Python\issue_01.py", line 68, in issue_01.encode
Failed example:
    encode('Аннушка уже разлила масло...') #doctest: +ELLIPSIS
Exception raised:
    Traceback (most recent call last):
      File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\doctest.py", line 1329, in __run
        compileflags, 1), test.globs)
      File "<doctest issue_01.encode[3]>", line 1, in <module>
        encode('Аннушка уже разлила масло...') #doctest: +ELLIPSIS
      File "D:\Python\issue_01.py", line 72, in encode
        LETTER_TO_MORSE[letter] for letter in message
      File "D:\Python\issue_01.py", line 72, in <listcomp>
        LETTER_TO_MORSE[letter] for letter in message
    KeyError: 'А'
1 items had no tests:
    issue_01
**********************************************************************
1 items had failures:
   1 of   4 in issue_01.encode
4 tests in 2 items.
3 passed and 1 failed.
***Test Failed*** 1 failures.



===================================================================================================================================
Test assert

D:\Python\venv\Scripts\python.exe D:/Python/issue_01.py
Traceback (most recent call last):
  File "D:/Python/issue_01.py", line 107, in <module>
    assert msg == enc, err_msg
AssertionError: -- .- .. -....- .--. -.-- - .... --- -. -....- ..--- ----- .---- ----. != .... .. --. .... .-.. --- .- -.. ..--..

Process finished with exit code 1
